contracts/universalSchemes/VoteInOrganizationScheme.sol

```diff
@@ -30,7 +30,7 @@ contract VoteInOrganizationScheme is UniversalScheme, ExecutableInterface, Actio
     }
 
     // A mapping from the organization (Avatar) address to the saved data of the organization:
-    mapping(address=>mapping(bytes32=>VoteProposal)) public organizationsData;
+    mapping(address=>mapping(bytes32=>VoteProposal)) public organizationProposals;
```

Comment:
organizationProposals - > organizationsProposals

Reply:
I'm not sure where this diff is coming from, it is named `organizationProposals` in the code.

Reply:
It should be name organizationsProposals

Reply:
ok, reverted back, except for the test code variable names